### ISSUE TITLE
tslint no longer working with vscode 0.10.10+

### ISSUE THREAD

**Issue by SeeeD** [Microsoft/vscode-tslint#34]
Since the last updates of vscode (0.10.10 and 0.10.11) tslint is no longer working. 
There is no error shown, but the rules are no longer working. 

Reinstalling of tslint did not resolve this issue. Can you investigate in this?

**Comment by larrifax** [comment]
Do you use custom rules? The plugin stops working for me when I enable any of my custom rules.

**Comment by SeeeD** [comment]
@larrifax Yes, I am using a tslint.json with custom rules in it.

**Comment by egamma** [comment]
We use tslint for our own development and it works. What version of:
- tslint
- vscode-tslint

![image](https://cloud.githubusercontent.com/assets/172399/13861705/136f36a2-ec8f-11e5-8031-1bcb111a56e7.png)

**Comment by egamma** [comment]
@larrifax did you configure the location of the custom rules in the user settings?

**Comment by larrifax** [comment]
@egamma Well, that's embarrasing.. I only did it in the ```gulp-tslint``` config and totally forgot about configuring the ```vscode-tslint``` plugin. It works when I add the rulesDirectory config to the VSCode settings.

On a side-note. When setting ```"tslint.rulesDirectory": ["node_modules/ng2lint/dist/src", "tools/tslint"]``` I get a squiggly-line saying that a string is expected. It totally works though.

**Comment by egamma** [comment]
@larrifax no problem, it would be better when you could do this configuration in a single place...

>On a side-note. When setting  "tslint.rulesDirectory": ["node_modules/ng2lint/dist/src", "tools/tslint"]  I get a squiggly-line saying that a string is expected. It totally works though.

This is buglet in the schema for settings. The schema for the settings needs to be updated to also support an array of strings. Filed #37 for this and closing this issue.

**Comment by egamma** [comment]
Closed too soon...

@SeeeD can you open the developer tools Help>Developer Tools and see whether there is an exception in the console.

Did you configure the rules in the settings see https://github.com/Microsoft/vscode-tslint/issues/34#issuecomment-198346348

**Comment by egamma** [comment]
Closing, this no longer seems to be an issue.
